snippets/Google.Logging.Log4Net.Snippets/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System.Reflection;
```

Comment:
We don't normally need an AssemblyInfo class for snippets, as far as I'm aware. (Will have to check.)